giskard_cicd/cli.py

```diff
@@ -126,9 +131,9 @@ def main():
     else:
         rendered_report = report.to_html()
 
-    if args.output_portal == "huggingface":
+    if args.output_portal == "huggingface" and args.leaderbord_dataset:
```

Comment:
I think the discussion could be published without `leaderboard_dataset`.

```suggestion
    if args.output_portal == "huggingface":
        # Create the discussion
        # ...
        if args.leaderboard_dataset:
            # Init dataset commit scheduler
            # ...
```

Reply:
Fixed